src/gauging.jl

```diff
@@ -73,114 +78,58 @@ function vidal_gauge(
   return ψ_vidal, bond_tensors
 end
 
-"""Use an ITensorNetwork ψ in the symmetric gauge and its mts to put ψ into the vidal gauge. Return the bond tensors and ψ_vidal."""
 function vidal_gauge(
   ψ::ITensorNetwork,
-  pψψ::PartitionedGraph,
-  mts;
-  eigen_message_tensor_cutoff=10 * eps(real(scalartype(ψ))),
-  regularization=10 * eps(real(scalartype(ψ))),
-  edges=NamedGraphs.edges(ψ),
-  svd_kwargs...,
+  bond_tensors;
+  (cache!)=nothing,
```

Comment:
Related to the discussion below, this function is confusing to me.

If we think about the Vidal gauge as the combination of site tensors `ψ` and the `bond_tensors`, why would there also be a message tensor cache being input into this function? Don't the `bond_tensors` play the role of the messages, in the Vidal gauge?